### PR TITLE
OCPBUGS-8003: Fix lost client subscription clean up and add validation 

### PR DESCRIPTION
**Pull request by aneeshkp**
Fix delete subscription on client dead

### CODE REVIEW

File: examples/manifests/kustomization.yaml [hunk 1]
```diff
@@ -11,8 +11,8 @@ replicas:
...
 - name: cloud-event-consumer
-  newName: quay.io/redhat-cne/cloud-event-consumer
-  newTag: latest
+  newName: quay.io/aneeshkp/cloud-event-consumer
+  newTag: 4.13-sub
 - name: cloud-event-sidecar
-  newName: quay.io/redhat-cne/cloud-event-proxy
-  newTag: latest
+  newName: quay.io/aneeshkp/cloud-event-proxy
+  newTag: 4.13-sub3
```
Comment (jzding) [inline examples/manifests/kustomization.yaml:18]: remove temporary changes.

Comment (aneeshkp) [reply examples/manifests/kustomization.yaml]: fixed . sorry,I forgot not to include this.

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**